src/api/java/com/minecolonies/api/util/constant/NbtTagConstants.java

```diff
@@ -562,6 +562,11 @@
     public static final String TAG_GUARD_NEARBY  = "guardnearby";
     public static final String TAG_RESPAWN_POS   = "nextrespawnpos";
 
+    /**
+     * Compound tag for the stack.
+     */
+    public static final String TAG_STACK = "stack_store";
```

Comment:
This is a different string from what it's replacing (which was just "stack")... is that going to cause persistence data loss?  Or is it only used in non-persistent contexts?

Reply:
It was never used before this change. ItemStorageFactory was effectively dead code before this change, as it wasn't referenced anywhere other than to register it, as far as I could find, and we don't persist ItemStorage anywhere that I can find. Racks, the most common usage of them store ItemStacks, and wrap them in ItemStorage for indexing purposes.